src/petTrack.js

```diff
@@ -0,0 +1,109 @@
+// PROTOTYPE SYNTAX PET
+
```

Comment:
Comments and clear lines like this at the top of a file are not needed, should be removed